packages/styled-ppx/styled-ppx.0.59.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis: "Type-safe styled components for ReScript and Melange"
+description:
+  "styled-ppx is the ppx that brings styled components to ReScript and Melange, allowing you to create React Components with type-safe style definitions using CSS."
+maintainer: ["David Sancho <dsnxmoreno@gmail.com>"]
+authors: ["David Sancho <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://styled-ppx.vercel.app"
+bug-reports: "https://github.com/davesnx/styled-ppx/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "5.1.0"}
+  "reason" {>= "3.11.0"}
+  "menhir" {>= "20220210"}
+  "ppx_deriving" {>= "5.0"}
+  "ppx_deriving_yojson" {>= "3.7.0"}
+  "ppxlib" {>= "0.27.0"}
+  "sedlex" {>= "3.2"}
+  "melange" {>= "3.0.0"}
+  "server-reason-react"
+  "reason-react" {>= "0.14.0"}
+  "alcotest" {with-test}
+  "conf-npm" {with-test}
+  "reason-react-ppx" {with-dev-setup}
+  "ocaml-lsp-server" {with-dev-setup}
+  "ocamlformat" {with-dev-setup}
+  "utop" {with-dev-setup}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
```
Tests are failing all over the place, but if I remember correctly they cannot work in the CI. Am I correct?

Reply:
It's very rare, I disabled some tests that relied on a npm dependency last release, but I don't understand why all cram tests do not work :/

I'm happy to disable the tests, since I run them on each PR/branch/tag on my repo

Reply:
I have no idea about the failure, I am not really familiar with `reason`, but indeed they still all look like the ones in 
https://github.com/ocaml/opam-repository/pull/26192#issuecomment-2216939296

Reply:
The only different one (which may have also been there in the past) is
```
# (cd _build/default/packages/runtime/test && ./test.exe)
# Testing `styled-ppx.native'.
# This run has ID `AUFX4VMP'.
# 
# .........F......................................................................
# .................
# 
# ┌──────────────────────────────────────────────────────────────────────────────┐
# │ [FAIL]        CSS                   9   style_tag.                           │
# └──────────────────────────────────────────────────────────────────────────────┘
# ASSERT should be equal
# FAIL should be equal
# 
#    Expected: `"<style data-reactroot=\"\" data-emotion=\"css 18zdck7 1gx3jof 13o7eu2\" data-s>html{line-height:1.15;} @keyframes animation-1gx3jof { 0% { -webkit-transform: rotate(0deg); -moz-transform: rotate(0deg); -ms-transform: rotate(0deg); transform: rotate(0deg); } 100% { -webkit-transform: rotate(-360deg); -moz-transform: rotate(-360deg); -ms-transform: rotate(-360deg); transform: rotate(-360deg); } } .css-13o7eu2 { display: block; }</style>"'
# 
#    Received: `"<style data-emotion=\"css 18zdck7 1gx3jof 13o7eu2\" data-s>html{line-height:1.15;} @keyframes animation-1gx3jof { 0% { -webkit-transform: rotate(0deg); -moz-transform: rotate(0deg); -ms-transform: rotate(0deg); transform: rotate(0deg); } 100% { -webkit-transform: rotate(-360deg); -moz-transform: rotate(-360deg); -ms-transform: rotate(-360deg); transform: rotate(-360deg); } } .css-13o7eu2 { display: block; }</style>"'
# 
# Raised at Alcotest_engine__Test.check in file "src/alcotest-engine/test.ml", lines 200-210, characters 4-19
# Called from Alcotest_engine__Core.Make.protect_test.(fun) in file "src/alcotest-engine/core.ml", line 181, characters 17-23
# Called from Alcotest_engine__Monad.Identity.catch in file "src/alcotest-engine/monad.ml", line 24, characters 31-35
# 
# Logs saved to `~/.opam/5.2/.opam-switch/build/styled-ppx.0.59.0/_build/default/packages/runtime/test/_build/_tests/styled-ppx.native/CSS.009.output'.
```

Reply:
Right, it's probably from another dep. It is publishing latest (https://github.com/ocaml/opam-repository/pull/26269) and updating here later

Reply:
Waiting for https://github.com/ocaml/opam-repository/pull/26275 to merge

Reply:
I fixing the issues in https://github.com/ocaml/opam-repository/pull/26276

Reply:
> I fixing the issues in https://github.com/ocaml/opam-repository/pull/26276

Does this mean you want to close this PR here? Or are both relevant?